README.md

```diff
@@ -88,7 +88,11 @@ commitsSince: One year ago
 filesToIgnore:
  - src/Commands/ChurnCommand.php
  - src/Results/ResultsParser.php
- ```
+
+# The math formula to calculate the score
+# Default: number of commits + the complexity of each files
+formula: '[[commits]] + [[complexity]]'
```

Comment:
Do you know if this is a standard way of putting placeholder values in a yml file (or if there even is a standard)?